plugins/rapid7_vulndb/help.md

```diff
@@ -137,12 +137,97 @@ _This plugin does not contain any triggers._
 |Published_at|string|False|Published date of vulnerability|
 |Title|string|False|Title of Vulnerability|
 
+#### Search Database
```

Comment:
@ablakley-r7 Since you are adding new actions, shouldn't this be 2.2.0 instead of 2.1.1?

Reply:
Error with regeneration caused duplicate action entries. Has been updated and resolved.